### DISCUSSION TITLE
Submenus for publications

### DISCUSSION THREAD

**Discussion by p-wegmueller** [alshedivat/al-folio#1706] [Q&A]
Dear community, 
I would like to set up submenus for my publications page, i.e., similar to http://info.phys.tsinghua.edu.cn/kimkihwan/ I would like to have something like 

 - publications
   --- academic
   --- policy
   --- working papers

I tried already quite a lot, but did not succeed. I would appreciate guidance on how to do that appropriately, also with corresponding .bib files. 
Which changes do I need to make in 
- config.yml
- layout.html
- publications.md

Thanks!

Currentyl, my page looks like: https://p-wegmueller.github.io/

**Comment by george-gca** [comment]
At least the submenu part is already supported in the current template. As for expand on hover, this probably is done via css, but I am not fluent in css. I would recommend checking their page source using the developer tools available in [Chrome](https://developer.chrome.com/docs/devtools/overview/) or [Firefox](https://firefox-source-docs.mozilla.org/devtools-user/).
